src/System.Private.ServiceModel/src/System/IdentityModel/Claims/ClaimSet.cs

```diff
@@ -48,7 +46,9 @@ public static ClaimSet Windows
                     s_windows = new DefaultClaimSet(claims);
                 }
                 return s_windows;
-#endif // FEATURE_NETNATIVE
+#else 
+                throw ExceptionHelper.PlatformNotSupported("Windows Stream Security is not supported on UWP yet");
+#endif // SUPPORTS_WINDOWSIDENTITY
```

Comment:
Your call on whether to make a new constant for it.  Could also be a helper method that returns a PNSE with that message in it.